src/org.openjdk.nashorn/share/classes/org/openjdk/nashorn/api/scripting/URLReader.java

```diff
@@ -82,7 +82,7 @@ public URLReader(final URL url, final Charset cs) {
     }
 
     @Override
-    public int read(final char cbuf[], final int off, final int len) throws IOException {
+    public int read(final char[] cbuf, final int off, final int len) throws IOException {
```

Comment:
Oops. Never knew we had those in the code :) 

Did you run any tool to find these or just found while code inspection/reading?

Reply:
IntelliJ IDEA warns about them. Most of this code changes were driven by IDEA highlighting things.